src/md/contributing.md

```diff
@@ -124,17 +145,15 @@ Thanks for contributing!
 
 ## Merging a Pull Request (committers)
```

Comment:
Should there also be a section on "Reviewing Pull Requests"? If nothing else, we can add a pointer to http://www.apache.org/foundation/voting.html

Reply:
Why a separate section?